mitodl/micromasters
# Added field for state or territory

#### What are the relevant tickets?
Fixes #229 

#### What's this PR do?
Adds a field to generate a list of states or territories based on the selected country

#### Where should the reviewer start?
`static/js/util/profile_edit.js`

#### How should this be manually tested?
Go to the personal tab and select a country. The currently selected state should be cleared and the user should be able to select from a list of territories for that country.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
